src/components/Products.js

```diff
@@ -34,7 +34,7 @@ export default function Products() {
       <div className="gallery">
         {isLoading && <h4>Loading products...</h4>}
         {getUniqueCategoryProducts.map((item) => (
```

Comment:
@pasternakewa possible error `getUniqueCategoryProducts` is it always exist?

Reply:
* do
  * ne

Reply:
wot?